templates/partials/language.mustache

```diff
@@ -13,18 +13,18 @@
       <span class="nav-caret"></span>
     </li>
     <li class="en">English</li>
+    <!--
     <li class="de">Deutsch</li>
+    -->
```

Comment:
Besides this list, you should remove Deutsch from the translation list 
https://nikolai-deriv.binary.sx/add-languages/bot.html?l=de